playbooks/roles/ocp-migration/tasks/sdn_to_ovnkube.yml

```diff
@@ -41,48 +41,56 @@
 
 
 # Start Migration
-- name: Set the migration field is null before setting it to a value
-  shell: |
-    oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec": {"migration": null}}'
 
 - name: To change default CNI network provider
   shell: |
     oc patch Network.operator.openshift.io cluster --type='merge' \
```

Comment:
Can we patch the Network using Ansible?

Reply:
@varad-ahirwadkar, any guidance on how to patch the Network using Ansible?

Reply:
Please try this code
```
- name: Change default CNI network provider
  kubernetes.core.k8s:
    state: present
    definition:
      apiVersion: config.openshift.io/v1
      kind: Network
      merge_type: merge
      metadata:
        name: cluster
      spec:
        migration: null
```

Reply:
Update code  to patch the Network using Ansible.